### DISCUSSION TITLE
Misalignment (overlapping) of text in epub files when the first letter is larger than the rest

### DISCUSSION THREAD

**Discussion by wrislab** [sumatrapdfreader/sumatrapdf#4538] [Q&A] [answered]
So, I tried opening the same file using other epub viewers, they viewed the text properly, except Sumatra. Point to be noted, honestly I love to read whether pdf or epub, using the sumatraPDF most. Thank you.

![image](https://github.com/user-attachments/assets/fb0e12b1-e715-4033-9ea2-97e4b9729fe8)

**Comment by kjk** [comment, marked as answer]
Does it happen in [pre-release](https://www.sumatrapdfreader.org/prerelease) ?

**Reply by wrislab** [reply]
Surprisingly, no! Thank you so much...

![image](https://github.com/user-attachments/assets/8e081f21-a8e8-4108-8f83-6affc37eb8fb)
A modification request, please consider if possible. Adding a bookmark shortcut on the toolbar. That's it. thank you so much again...

Edit: Just noticed the texts and signs on the tool bar in dark mode. My goodness. Can't thank you enough!

**Reply by wrislab** [reply]
When I click right button on mouse pressing Ctrl to lookup words on WordWeb, the application crushes.

**Comment by GitHubRulesOK** [comment]
Request perhaps possible ?
>a bookmark shortcut on the toolbar.

IF there is a suitable command you can add your own toolbar "iconic text" :-)

see pre-release adds extra abilities for redirecting command shortcuts but bookmarks is basically just `CmdToggleBookmarks` so a text icon for that is possible ? Say `Bkm`

![image](https://github.com/user-attachments/assets/4bcb975a-07de-4456-b4bb-d0a454dcd956)


```
Shortcuts [
	[
		Cmd = CmdToggleBookmarks
		Key = x
		ToolbarText = Bkm
	]
]
```
